satellite/audit/queue.go

```diff
@@ -0,0 +1,45 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package audit
+
+import (
+	"sync"
+
+	"github.com/zeebo/errs"
+
+	"storj.io/storj/pkg/storj"
+)
+
+// ErrEmptyQueue is used to indicate that the queue is empty.
+var ErrEmptyQueue = errs.Class("empty audit queue")
+
+// Queue is a list of paths to audit, shared between the reservoir chore and audit workers.
+type Queue struct {
+	mu    sync.Mutex
+	Queue []storj.Path
```

Comment:
Try to avoid exposing the internal state of the queue. It's used concurrently, so it probably shouldn't be accessed outside the implementation.